<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>COMP_hack Cathedral of Content</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <widget class="QGroupBox" name="groupZone">
      <property name="title">
       <string>Zone</string>
      </property>
      <layout class="QVBoxLayout" name="zoneLayout">
       <item>
        <layout class="QHBoxLayout" name="zoneCurrentLayout">
         <item>
          <widget class="QLabel" name="zonePath">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>No file is open.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="zoneBrowse">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="zoneView">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>View Zone</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupEvent">
      <property name="title">
       <string>Event</string>
      </property>
      <layout class="QHBoxLayout" name="eventLayout">
       <item>
        <widget class="QLabel" name="eventCount">
         <property name="text">
          <string>No events have been loaded.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="eventsView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>View Events</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupDropSets">
      <property name="title">
       <string>Drop Set</string>
      </property>
      <layout class="QHBoxLayout" name="dropSetLayout">
       <item>
        <widget class="QLabel" name="dropSetCount">
         <property name="text">
          <string>No drop sets have been loaded.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="dropSetView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>View Drop Sets</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="logGroup">
      <property name="title">
       <string>Log</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextEdit" name="log">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>695</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuResourceList">
     <property name="title">
      <string>Resource List</string>
     </property>
    </widget>
    <addaction name="menuResourceList"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
